vowpalwabbit/cbify.cc

```diff
@@ -745,7 +745,7 @@ base_learner* cbify_setup(VW::setup_base_i& stack_builder)
     }
   }
 
-  if (data->use_adf)
+  if (data->use_adf && (options.was_supplied("regcb") || options.was_supplied("squarecb")))
```

Comment:
Why gateway on regcb or squarecb?  It seems like the ability to encode losses should be independent of the chosen algorithm.

Reply:
The reason for this gateway is that supplying `cb_min_cost` when no reduction actually consumes the option will produce an error. By checking that either one of those is in use we avoid showing a warning to a user that they did not cause